main.lua

```diff
@@ -347,8 +349,12 @@ mod:AddCallback(ModCallbacks.MC_POST_NEW_LEVEL, mod.updatePlanetariumChance)
 mod:AddCallback(ModCallbacks.MC_POST_GAME_STARTED, mod.init)
 mod:AddCallback(ModCallbacks.MC_PRE_GAME_EXIT, mod.exit)
 
-mod:AddCallback(ModCallbacks.MC_GET_SHADER_PARAMS, mod.onRender)
-mod:AddCallback(ModCallbacks.MC_POST_RENDER, mod.onRender)
+if REPENTOGON then
```

Comment:
The REPENTOGON variable is defined by repentogon itself, much like how the REPENTANCE variable is defined by the game itself. Any other mod that changes it will break a lot more than planetarium chance, so it's a non-issue that's not worth worrying about.

Reply:
Additionally, even if it was something different than expected, it'd just run the vanilla code which still works perfectly fine with repentogon installed.